addons/base/views.py

```diff
@@ -289,6 +289,7 @@ def get_auth(auth, **kwargs):
             'callback_url': node.api_url_for(
                 ('create_waterbutler_log' if not node.is_registration else 'registration_callbacks'),
                 _absolute=True,
+                _internal=settings.DEV_MODE,
```

Comment:
same question as below "can you elaborate on why is this correct only in DEV_MODE?"